.pre-commit-config.yaml

```
@@ -0,0 +1,7 @@
+repos:
+  -   repo: https://github.com/doublify/pre-commit-rust
+      rev: master
+      hooks:
+        -   id: fmt
+        -   id: cargo-check
+        -   id: clippy
```

The formatting here is a bit funky, isn't it?
```suggestion
repos:
  - repo: https://github.com/doublify/pre-commit-rust
    rev: master
    hooks:
      - id: fmt
      - id: cargo-check
      - id: clippy
```